planner/core/common_plans.go

```diff
@@ -529,10 +529,12 @@ func (e *Explain) prepareOperatorInfo(p PhysicalPlan, taskType string, indent st
 	row := []string{e.prettyIdentifier(p.ExplainID(), indent, isLastChild), count, taskType, operatorInfo}
 	if e.Analyze {
 		runtimeStatsColl := e.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl
-		if runtimeStatsColl.Exists(p.ExplainID()) {
-			row = append(row, runtimeStatsColl.Get(p.ExplainID()).String())
-		} else {
-			row = append(row, "") //TODO: wait collect more executor info from tikv
+		// There maybe some mock information for cop task to let runtimeStatsColl.Exists(p.ExplainID()) is true.
```

Comment:
Will the case mentioned in this comment happen for now?

Reply:
Yes.
The root task stats are created at `newBaseExecutor`, and it is unknown weather the executor is built by a cop task or a root task, so the default action now is creating a root stats for it.

Maybe we can introduce one more parameter into `newBaseExecutor` to indicate the type of task, but I prefer to do it in another PR because this PR is big enough...

Reply:
Can this comment be removed after implementing analyze in TiKV?

Reply:
I'm afraid not.
For each cop task, an empty root stats will be created for it in `newBaseExecutor`.
So unless we fix it in `newBaseExecutor`, this problem will always exist.

Reply:
Do you mean rootStatsMap and copStatsMap will contain the same explainID?

Reply:
yes...